src/client/board/Board.js

```diff
@@ -21,20 +24,30 @@ export const Board = () => {
     }
 
     const boardStyles = {
-        background: "linear-gradient(135deg," + theme.base + ", transparent)"
+        background: 'linear-gradient(135deg,' + theme.base + ', transparent)'
```

Comment:
Agree. But how can I send a color to styles if it depends on theme?

Reply:
You can create several classes with different colors. 
As I understand you don't have any custom color theme there, do you?

Reply:
Yes, all themes are predefined. But these colors are used in many different places. What if I need to change some theme color? I would have to go to all of these stylesheet files and change it?

Reply:
I guess there're two solutions:
* use postcss and postcss-css-variables packages to have css variables which will contain colors
* create classes for each type of color only (it's not really BEM, but I don't see other options to handle colors in one place), like:
.blue-text {
  color: blue;
}
.blue-background {
  backgroud-color: blue;
}
and then use them in js like `className = "board-body blue-background"`

Otherwise, you can create classes with color modificators, but you would need to change color in many places in case of some changes

Reply:
What about some compromise?
I set up custom properties with theme-colors in Board component and use these properties in other places. So there are no inline styles anywhere except Board component. Please, take a look. Commit 5950a8d3624d000476e0f621c2567e3cbcf1479a

Reply:
@alex-zh, please, take a look